src/main/java/ua/com/juja/microservices/teams/service/Utils.java

```diff
@@ -0,0 +1,20 @@
+package ua.com.juja.microservices.teams.service;
+
+import java.io.IOException;
+import java.io.Reader;
+
+/**
+ * @author Ivan Shapovalov
+ */
+public class Utils {
+
+    public static String convertToString(Reader reader) throws IOException {
```

Comment:
If you use this method only in the tests you should replace Utils class to the test folder.